lib/environment_setup.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 # rubocop:disable Rails/Output
+
+require "dotenv"
+Dotenv.load(".ruby-env")
```

Comment:
Yeh I'am down to talk about this decision. I decided to use this because my computer doesn't use rvm to control the version of ruby I'am using. I wanted to be able to load the environment variables stored in `.ruby-env` without having to use rvm. One solution is to use `dotenv` library which is an application level gem that is agnostic to your ruby version manager.